### PR TITLE
chore: Added spotless and explicit toString() impls

### PR DESCRIPTION
**Pull request by singhalprerana**
Added spotless plugin which resulted in formatting of all files..
Also, added explicit `toString()` impl for `RequestContext` and `DefaultJwt` classes

### CODE REVIEW

File: grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java [hunk 1]
```diff
@@ -138,4 +138,15 @@ public ContextualKey<Void> buildInternalContextualKey() {
   public <T> ContextualKey<T> buildInternalContextualKey(T data) {
     return new DefaultContextualKey<>(this, data, List.of(TENANT_ID_HEADER_KEY));
   }
+
+  @Override
```
Comment (aaron-steinfeld) [inline grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java:142]: nit: we typically use lombok so it gets updated automatically

File: grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java [hunk 2]
```diff
@@ -138,4 +138,15 @@ public ContextualKey<Void> buildInternalContextualKey() {
...
+  public String toString() {
+    final String emptyValue = "{}";
+    return "RequestContext{"
+        + "headers="
+        + headers
+        + ", jwt="
+        + getJwt().map(Jwt::toString).orElse(emptyValue)
```
Comment (aaron-steinfeld) [inline grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java:149]: a bit duplicative but I suppose it makes it easier

File: grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java (hunk 1 above)
Comment (singhalprerana) [reply grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java:142]: but doesn't lombok use the class members to generate the string? this is kind of specific to what we want - not the class members as such..

File: grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java (hunk 2 above)
Comment (singhalprerana) [reply grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java:149]: I assumed we would want to see jwt details too

File: grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java (hunk 1 above)
Comment (aaron-steinfeld) [reply grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java:142]: It's configurable. Can mark getter methods if you want I believe (and most of the jwt details are already exposed as getters)

**Review by aaron-steinfeld — APPROVED** [review]

**Review by tim-mwangi — APPROVED** [review]

File: grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java (hunk 1 above)
Comment (singhalprerana) [reply grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java:142]: oh saw ur msg after merging - I guess should be ok for now...

**Pull request merged**